### PR TITLE
CONTENTBOX-831 : FileBrowser - Filter to only show directories for As…

### PR DESCRIPTION
**Pull request by cfmitrah**
…set Chooser

Added Filter for show directories

### CODE REVIEW

**Comment by lmajano** [comment]
WOW, awesome!! Thanks so much @cfmitrah  will strt reviewing

File: modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc [hunk 1]
```diff
@@ -540,6 +557,10 @@ component{
 				prefs.listType = "listing";
 				cookieStorage.setVar( "fileBrowserPrefs", serializeJSON( prefs ) );
 			}
+			if( !structKeyExists( prefs, "listFolder" ) ){
+				prefs.listType = "listing";
```
Comment (lmajano) [inline modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc]: Shouldn't this be `prefs.listFolder`?

File: modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc [hunk 2]
```diff
@@ -110,6 +110,23 @@ component{
 		// get directory listing.
 		prc.fbqListing = directoryList( prc.fbCurrentRoot, false, "query", prc.fbSettings.extensionFilter, "#prc.fbPreferences.sorting#" );
 
+		if(prc.fbPreferences.listFolder eq "dir" and structkeyexists(rc,"sorting")){
+			if(prc.fbPreferences.sorting eq "lastmodified"){
+				prc.fbPreferences.sorting ="datelastmodified"
+			}
+			var fileListQuery = new query();
```
Comment (lmajano) [inline modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc]: Can we update this to be a litte more streamline:

```
var fileListQuery = new Query( 
	dbType = "query",
	qry = prc.fbqListing,
	sql = "select * from qry where type=:dir order by #prc.fbPreferences.sorting#"
);

fileListQuery.addParam( name="dir", value="dir" );
```

Comment (cfmitrah) [reply modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc]: @lmajano I have updated the codes.can you Please review it...

**Comment by cfmitrah** [comment]
@lmajano Can You Please review This and let me know if any more changes needed...

**Pull request merged**